zhmcclient/_resource.py

```diff
@@ -174,3 +174,29 @@ def get_property(self, name):
                 raise
             self.pull_full_properties()
             return self._properties[name]
+
+    def __str__(self):
+        """
+        Convert a BaseResource object to a string.
+        Used for printing.
```

Comment:
"Used for printing" does not really define the purpose. "Used for human consumption" would be better, because that makes it clear that users are not supposed to parse the result.